README.md

```diff
@@ -100,3 +104,7 @@ Please refer to our [Code of Conduct][codeconduct].
 [codeconduct]: https://github.com/dragonflyoss/community/blob/master/CODE_OF_CONDUCT.md
 [d7y.io]: https://d7y.io/
 [dingtalk]: docs/images/community/dingtalk-group.jpeg
+
+## Software bill of materials
```

Comment:
Remove this part.

Reply:
Added information on Software Bill of Materials (SBOM)